include/zephyr/bluetooth/iso.h

```diff
@@ -690,195 +724,202 @@ struct bt_iso_accept_info {
 #if defined(CONFIG_BT_SMP) || defined(__DOXYGEN__)
-	/** Required minimum security level.
+	/**
+	 * @brief Required minimum security level.
+	 *
 	 * Only available when @kconfig{CONFIG_BT_SMP} is enabled.
 	 */
 	bt_security_t		sec_level;
 #endif /* CONFIG_BT_SMP */
 
-	/** @brief Server accept callback
+	/**
+	 * @brief Server accept callback
 	 *
-	 *  This callback is called whenever a new incoming connection requires
-	 *  authorization.
+	 * This callback is called whenever a new incoming connection requires authorization.
 	 *
-	 *  @param info The ISO accept information structure
-	 *  @param chan Pointer to receive the allocated channel
+	 * @param info The ISO accept information structure
+	 * @param chan Pointer to receive the allocated channel
 	 *
-	 *  @return 0 in case of success or negative value in case of error.
+	 * @return 0 in case of success or negative value in case of error.
 	 */
-	int (*accept)(const struct bt_iso_accept_info *info,
-		      struct bt_iso_chan **chan);
+	int (*accept)(const struct bt_iso_accept_info *info, struct bt_iso_chan **chan);
 };
 
-/** @brief Register ISO server.
+/**
+ * @brief Register ISO server.
  *
- *  Register ISO server, each new connection is authorized using the accept()
- *  callback which in case of success shall allocate the channel structure
- *  to be used by the new connection.
+ * Register ISO server, each new connection is authorized using the accept()
+ * callback which in case of success shall allocate the channel structure
+ * to be used by the new connection.
  *
- *  @param server Server structure.
+ * @param server Server structure.
  *
- *  @return 0 in case of success or negative value in case of error.
+ * @return 0 in case of success or negative value in case of error.
  */
 int bt_iso_server_register(struct bt_iso_server *server);
 
-/** @brief Unregister ISO server.
+/**
+ * @brief Unregister ISO server.
  *
- *  Unregister previously registered ISO server.
+ * Unregister previously registered ISO server.
  *
- *  @param server Server structure.
+ * @param server Server structure.
  *
- *  @return 0 in case of success or negative value in case of error.
+ * @return 0 in case of success or negative value in case of error.
  */
 int bt_iso_server_unregister(struct bt_iso_server *server);
 
-/** @brief Creates a CIG as a central
+/**
+ * @brief Creates a CIG as a central
  *
- *  This can called at any time, even before connecting to a remote device.
- *  This must be called before any connected isochronous stream (CIS) channel
- *  can be connected.
+ * This can called at any time, even before connecting to a remote device.
+ * This must be called before any connected isochronous stream (CIS) channel can be connected.
  *
- *  Once a CIG is created, the channels supplied in the @p param can be
- *  connected using bt_iso_chan_connect.
+ * Once a CIG is created, the channels supplied in the @p param can be
+ * connected using bt_iso_chan_connect().
```

Comment:
```suggestion
 * connected using @ref bt_iso_chan_connect().
```

Reply:
@ref is implied when ending the function with `()`

Reply:
My bad I didn't knew.